qt_ui/windows/settings/QSettingsWindow.py

```diff
@@ -705,7 +705,7 @@ def initGeneratorLayout(self):
 
         start_type_label = QLabel(
             "Default start type for AI aircraft<br /><strong>Warning: "
-            "Any option other than Cold breaks OCA/Aircraft missions.</strong>"
+            "Options other than Cold reduce utility of OCA/Aircraft missions.</strong>"
```

Comment:
```suggestion
            "Options other than Cold will significantly reduce the number of targets available for OCA/Aircraft missions, and OCA/Aircraft flights will not be included in automatically planned OCA packages.</strong>"
```

(add `<br />` as needed so it doesn't stretch the window weirdly)